src/assignment02/Assignment02.py

```diff
@@ -0,0 +1,86 @@
+#-----------------------------------------------------------------------------#
+# Title: Assignment02 - simple method of assignment (yes tuples)
+# Desc: This script is an automated interviewer using asking question
+# of the user and providing feedback based on that a while storing questions
+# and sessions in tuples
+# Change Log:
+#   Andrew Hopkins, 5/13/2024, created script
+#-----------------------------------------------------------------------------#
+
+Q1: str = "What is your name?"
+Q2: str = "What is your conference ID?"
+Q3: str = "Which organization do you represent?"
+Q4: str = "What is your email address?"
+Q5: str = "Please state any food preferences you have"
+listofqs: tuple = (Q1, Q2, Q3, Q4, Q5)
+
+Ses1: str = "Python for Beginners"
```

Comment:
Same comments for the variables in this file - should be snake_case, session_one for example